src/libsystemd/sd-journal/mmap-cache.h

```diff
@@ -1,23 +1,35 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
+#include <errno.h>
 #include <stdbool.h>
 #include <sys/stat.h>
 
-/* One context per object type, plus one of the header, plus one "additional" one */
-#define MMAP_CACHE_MAX_CONTEXTS 9
-
 typedef struct MMapCache MMapCache;
 typedef struct MMapFileDescriptor MMapFileDescriptor;
 
+typedef enum MMapCacheContext {
```

Comment:
usually we call context objects "context". Maybe use a differen word here, maybe "indicator", or "designator" or "class" or "category" or so